support-frontend/assets/components/footerCompliant/BackToTop.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { css } from '@emotion/core';
+import {
+  brandBackground,
+  brandText,
+  brandAlt,
+} from '@guardian/src-foundations/palette';
+import { textSans } from '@guardian/src-foundations/typography';
+
+const iconHeight = '42px';
+
+const iconContainer = css`
+    position: relative;
+    float: right;
+    border-radius: 100%;
+    background-color: ${brandBackground.ctaPrimary};
+    cursor: pointer;
+    height: ${iconHeight};
+    min-width: ${iconHeight};
+`;
+
+const icon = css`
+    :before {
+        position: absolute;
+        top: 6px;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        margin: auto;
+        border: 2px solid ${brandText.ctaPrimary};
+        border-bottom: 0;
+        border-right: 0;
+        content: '';
+        height: 12px;
+        width: 12px;
+        transform: rotate(45deg);
+    }
+`;
+
+const link = css`
+    text-decoration: none;
+    color: ${brandText.anchorPrimary};
+    font-weight: bold;
+    line-height: ${iconHeight};
+    :hover {
+      color: ${brandAlt[400]};
+      text-decoration: none;
+      & .css-${iconContainer.name} {
+        background-color: ${brandAlt[400]};
+      }
+    }
+`;
+
+
+const textStyles = css`
+    ${textSans.small({ fontWeight: 'bold' })};
```

Comment:
I've noticed issues with passing fontWeight into options in the past so generally I separate it out. If you've tested across a few browsers and it works though feel free to ignore.

Reply:
Oh, are you able to characterise those issues @rupertbates?

Reply:
Yes the bold wasn't getting applied

Reply:
I've checked in Firefox, Chrome and Safari locally and had a look in IE too via Browserstack and it all looked fine. Might be an issue with specific fonts? If the weight you specify isn't in the font file(s) downloaded then browsers will usually just default to the nearest one they have.

Reply:
Interesting! @i-hardy makes a good point. Here are the [available font wieghts for each font](https://www.theguardian.design/2a1e5182b/p/95d5d0-code/t/98bef6).

Reply:
Ah, maybe that's the problem. I think I've mostly just been using 'bold' though...